core/jobs/src/test/java/com/cognifide/aet/job/common/modifiers/click/ClickModifierTest.java

```diff
@@ -1,14 +1,14 @@
 /**
  * Automated Exploratory Tests
- *
+ * <p>
```

Comment:
Well some of them do have... (eg ValidatorException class), it should be fixed by global reformat (in separeate task)